.github/workflows/gotest.yml

```diff
@@ -0,0 +1,51 @@
+on: [push, pull_request]
+name: Go test
+jobs:
+  oldest:
+    name: The oldest possible Go version whatever hacks needed
+    runs-on: ubuntu-16.04
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.6.x
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Hack for ancient mod-incompatible Go versions
+      run: mkdir -p /home/runner/go/src/github.com/apsystole && cp -pr . /home/runner/go/src/github.com/apsystole/log
```

Comment:
For some reason this `mkdir -p` returns `Operation not permitted` on macos-latest image. On ubuntu it works. On both platforms, the symlink is not understood by golang, hence I just copy the files over.